### PR TITLE
Update build

### PR DESCRIPTION
**Pull request by mpetruska**
* [x] ready for review/merge

### CODE REVIEW

**Review by solidsnakedev — COMMENTED** [review]
Let's add this to .gitignore

File: .pre-commit-config.yaml
Comment (solidsnakedev) [inline .pre-commit-config.yaml:1]: let's add this to gitignore

File: .vscode/tasks.json
Comment (solidsnakedev) [inline .vscode/tasks.json:1]: what's this for?

File: app/Main.hs [hunk 1]
```diff
@@ -63,7 +69,22 @@ writePlutusScript title filepath term = do
...
+
+  setSGR [Reset]
+  writePlutusScript "Smart Handle Router" "./compiled/smartHandle.plutus" $
+    psmartHandleValidatorW
+      # ( pcon $
+            PSmartConfig $
+              pdcons @"cs"
+                # pdata padaSymbol
+                #$ pdcons @"swapScript"
+                # pdata (pconstant $ scriptHashAddress "11a65ca58a4e9c755fa830173d2a5caed458ac0c73f97db7faae2e7e3b52563c5410bff6a0d43ccebb7c37e1f69f5eb260552521adff33b9c2")
```
Comment (solidsnakedev) [inline app/Main.hs:83]: We don't need to apply parameters here, let's apply them with the tx builder

File: .vscode/tasks.json
Comment (mpetruska) [reply .vscode/tasks.json:1]: This configures the tasks and default tasks for VSCode. It might be a personal preference, so I'm adding this to .gitignore.

**Review by solidsnakedev — APPROVED** [review]

**Pull request merged**